docs/source/development/add-ons/create-an-add-on-prerelease.md

```diff
@@ -0,0 +1,197 @@
+---
+myst:
+  html_meta:
+    "description": "How to create an add-on for Volto 18 (development or pre-release)"
+    "property=og:description": "How to create an add-on for Volto 18 (development or pre-release)"
+    "property=og:title": "Create an add-on for Volto 18 (development or pre-release)"
+    "keywords": "add-on, Volto, create, development, pre-release"
+---
+
+# Create an add-on for Volto 18 (development or pre-release)
+
+This chapter describes how you can create a Volto add-on using the latest **development release** version of Plone with **Volto 18 or later** for the user interface, while having full control over its development and deployment.
+
+```{versionadded} Volto 18.0.0-alpha.43
+{term}`Cookieplone` is now the method to create a Plone add-on with pre-release versions of Volto, version 18.0.0-alpha.43 and above.
+```
+
+## System requirements
+
+Follow the section {ref}`plone:create-project-cookieplone-system-requirements` to set up your system.
+
+
+## Generate the add-on project
+
+Run the following command to generate your add-on project using the `frontend_addon` Cookieplone template.
```

Comment:
We should have at least a note that the project template might be a better option if you need both a frontend and backend addon that work together.

Reply:
What's the command for that? The readmes in both the cookieplone and cookieplone-templates are not clear. Or should we instead tell users to run `pipx run cookieplone` without specifying a template, and have them select one option accordingly?

Reply:
`pipx run cookieplone project`?

Reply:
Creating a full project with frontend and backend is already covered in https://6.docs.plone.org/install/create-project-cookieplone.html (you can use the same project to build an add-on). This chapter should primarily cover the frontend_addon template, and just make sure to give the reader a note that there is another option.

Reply:
(yeah it would be `pipx run cookieplone project`. I didn't realize it supports passing the template name on the command line like that.)